locust/webui/src/utils/object.ts

```diff
@@ -0,0 +1,36 @@
+export const isEmpty = <T extends Record<string, any>>(object: T) => !Object.keys(object).length;
+
+export function merge<ObjectA, ObjectB>(objectA: ObjectA, objectB: ObjectB) {
```

Comment:
+1 for just using Lodash

Reply:
I think lodash may be a little overkill for this project, especially since there is never really a case that we need a deep merge. We could even use JavaScript's built-in `Object.merge`